kolibri/content/utils/content_types_tools.py

```diff
@@ -1,15 +1,19 @@
 from django.db.models import Q
 from le_utils.constants import content_kinds
 
-from kolibri.content.hooks import ContentRendererHook
+from ..hooks import ContentRendererHook
+from ..models import ContentNode
 
-# Regardless of which renderers are installed, we can render topics!
-renderable_contentnodes_q_filter = Q(kind=content_kinds.TOPIC)
+# Start with an empty queryset, as we'll be using OR to add conditions
+renderable_contentnodes_without_topics_q_filter = ContentNode.objects.none()
```

Comment:
In many places where we used `renderable_contentnodes_q_filter`, we were then filtering out topics. Sped things up a bit by having a version of it that didn't include topics to start with.